makefiles/libc/newlib.mk

```diff
@@ -13,7 +13,12 @@ ifeq (1,$(USE_NEWLIB_NANO))
   export LINKFLAGS += -specs=nano.specs
 endif
 
-export LINKFLAGS += -lc -lnosys
+
+ifeq ($(TARGET_ARCH),mips-mti-elf)
+  export LINKFLAGS += -lc
+else
+  export LINKFLAGS += -lc -lnosys
```

Comment:
hmm, I thought this had worked, but must have missed the error message.
jerryscript is not compiling on arm targets without nosys:

/usr/bin/../lib/gcc/arm-none-eabi/6.3.1/../../../../arm-none-eabi/lib/thumb/v6-m/libc_nano.a(lib_a-gettimeofdayr.o): In function `_gettimeofday_r':
gettimeofdayr.c:(.text._gettimeofday_r+0xe): undefined reference to `_gettimeofday'